### PR TITLE
Tania user model test

### PR DESCRIPTION
**Pull request by adamgyulavari**

### CODE REVIEW

File: selection/spec/factories/applicant.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    first_name "John"
+    last_name "Doe"
+    email_address "John@Doe.com"
+    phone_number  "0123456789"
+    highest_education "BA"
+    english_knowledge "Basic"
+    english_class "yes"
+    payment_option "prepaid"
+  end
+end
```
Comment (adamgyulavari) [inline selection/spec/factories/applicant.rb]: new line

File: selection/spec/models/applicant_spec.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+
+RSpec.describe Applicant, type: :model do
+  it 'has a valid factory' do
+    expect(create :applicant).to be_valid
```
Comment (adamgyulavari) [inline selection/spec/models/applicant_spec.rb]: @faklijta instead of `create` use `build`

File: selection/spec/factories/applicant.rb (hunk 1 above)
Comment (faklijta) [reply selection/spec/factories/applicant.rb]: done

File: selection/spec/models/applicant_spec.rb (hunk 2 above)
Comment (faklijta) [reply selection/spec/models/applicant_spec.rb]: done

**Pull request merged**